memecity.game/Game/Components/ItemComponent.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _ITEMCOMPONENT_H
+#define  _ITEMCOMPONENT_H
+#include <ECS.h>
+#include <iostream>
+#include <Engine\MultimediaManager.h>
+
+struct ItemComponent : public memecity::engine::ecs::Component
+{
+	std::string name, description;
+
+	int damage, defence;
+
+	ItemComponent(memecity::engine::ecs::Entity& entity) : ItemComponent(entity, "unkown", "no description") {};
```

Comment:
We willen niet dat er een item met deze argumenten bestaat. Deze constructor laat foutgevoeligheid toe door argumentloze constructie toe te laten.